spec.bs

```diff
@@ -2495,6 +2495,26 @@ container/fenced navigable=] will fail, as outlined in [[#navigation-patch]].
 
 </div>
 
+<h4 id=allow-automatic-beacons>The
+\`<a http-header><code>Allow-Fenced-Frame-Automatic-Beacons</code></a>\` HTTP response header</h4>
+
+Serving a document [=resource=] that gets loaded into a {{Document}} that is not [=same origin=]
+with its [=Document/browsing context=]'s [=fenced frame config instance=]'s [=fenced frame config
+instance/mapped url=] with the <dfn
+http-header><code>Allow-Fenced-Frame-Automatic-Beacons</code></dfn> HTTP response header can
```

Comment:
"can"? or does? https://infra.spec.whatwg.org/#conformance. Same with the other header section below.

Reply:
Removed "can" from the 2 sections.